index.md

```diff
@@ -36,47 +36,46 @@ Thinking about using Anbox Cloud for your next project? [Get in touch!](https://
 | 0 | / | [Anbox Cloud documentation](https://discourse.ubuntu.com/t/anbox-cloud-documentation/17029) |
 | 0 | | |
 | 1 | tut/landing | [Tutorials](https://discourse.ubuntu.com/t/tutorials/28826) |
-| 2 | tut/installing-appliance | [Install the appliance](https://discourse.ubuntu.com/t/install-appliance/22681) |
-| 2 | tut/getting-started-dashboard | [Get started (web dashboard)](https://discourse.ubuntu.com/t/getting-started-with-anbox-cloud-web-dashboard/24958)|
-| 2 | tut/getting-started | [Get started (CLI)](https://discourse.ubuntu.com/t/getting-started/17756)|
-| 2 | tut/stream-client | [Set up a stream client](https://discourse.ubuntu.com/t/set-up-a-stream-client/37328) |
-| 2 | tut/creating-addon | [Create an addon](https://discourse.ubuntu.com/t/creating-an-addon/25284)|
+| 2 | tut/installing-appliance | [1.Install the appliance](https://discourse.ubuntu.com/t/install-appliance/22681) |
+| 2 | tut/getting-started-dashboard | [2.Get started using web dashboard](https://discourse.ubuntu.com/t/getting-started-with-anbox-cloud-web-dashboard/24958)|
+| 2 | tut/getting-started | [3.Get started using CLI](https://discourse.ubuntu.com/t/getting-started/17756)|
```

Comment:
Shouldn't we prepend with an article?
```suggestion
| 2 | tut/getting-started-dashboard | [2.Get started using the web dashboard](https://discourse.ubuntu.com/t/getting-started-with-anbox-cloud-web-dashboard/24958)|
| 2 | tut/getting-started | [3.Get started using the CLI](https://discourse.ubuntu.com/t/getting-started/17756)|
```
What  do you think?

Reply:
I was worried about the navigation bar spacing but let's try it. We can revert it if it looks ugly on the site.